CHANGELOG.md

```diff
@@ -3570,7 +3579,9 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 
 ## [v1.0.0] - 2018-02-23
 
-[unreleased]: https://github.com/nasa/cumulus/compare/v2.0.1...HEAD
+[unreleased]: https://github.com/nasa/cumulus/compare/v3.0.0...HEAD
+[v3.0.0]: https://github.com/nasa/cumulus/compare/v2.0.1...v3.0.0
+[v2.0.6]: https://github.com/nasa/cumulus/compare/v2.0.1...v2.0.7
```

Comment:
Are these valid? We want to show the diff between 2.0.1 and 2.0.7, not between 2.0.6 and 2.0.7? Also do we want to add the link for 2.0.7?

Reply:
Good catch, that should be 2.0.7, I'll fix the typo.

Reply:
091e6c9b8 re the version typo

Reply:
re: the range from 2.0.1, that's because the range in *master* is from tag 2.0.1 (the last release merged back to master) to the release tag, including all backport fixes.         We've done that for the last few backports - it's really a convenience link, if we think users looking at backports are going to want a per version changeset while looking at master, we could do otherwise.

Reply:
I went ahead and updated this to have the 2.0.x links map to the prior version, left the 3.0.0 referencing the prior release on main.  de0099f51